### DISCUSSION TITLE
Handle subscriptions over different connections to different servers

### DISCUSSION THREAD

**Discussion by alejandro2196** [FreeOpcUa/opcua-asyncio#1132] [Q&A]
Hello,

Is it possible to handle subscriptions over different connections to different servers?

I have a list with different connections, the subscriptions on the first one work correctly, but the subscriptions to the second server do not enter the handler.

Thanks for your time.

**Comment by AndreasHeine** [comment]
each connection to a server is a own client instance with a own subscription!

**Comment by alejandro2196** [comment]
This is how I have it... a list with client instances for each server, but it doesn't work.

Knowing that it is possible I will continue trying, thank you!
